portfolio/src/main/java/com/google/sps/servlets/GlobalWarmingDataServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns global warming sentiment data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
```

Comment:
{"2017": 52, "2018": 34} doesn't look like the data in .csv file?

Reply:
I had based this code on the Google Charts tutorial for this week, and forgot to modify part of the comment I had based this on. I'll update it for accuracy.